unity/Assets/Scripts/CanOpen_Object.cs

```diff
@@ -22,11 +20,12 @@ public class CanOpen_Object : MonoBehaviour
     public float animationTime = 0.2f;
 
     [SerializeField]
-    public float currentOpenPercentage = 1.0f; //0.0 to 1.0 - percent of openPosition the object opens. 
+    public float currentOpenPercentage = 1.0f; // 0.0 to 1.0 - percent of openPosition the object opens. 
+    private float startOpenness; // used to reset on failure
```

Comment:
I'm fine with whatever you choose, but can you make the naming consistent.  Use say "currentOpenness" and "startOpenness" or "currentOpenPercentage" and "startOpenPercentage"

Reply:
Yeah, that's fair. I was just using `currentOpenPercentage` as the legacy variable name, while `startOpenness` as the new name.